src/Lesson2/ImmutableClassTask/ImmutableGirl.java

```diff
@@ -0,0 +1,33 @@
+package Lesson2.ImmutableClassTask;
+
+public class ImmutableGirl {
+    private final int id;
+    private final String name;
+    private final Age age;
+
+    public ImmutableGirl (int id, String name, Age age) {
+        this.name = name;
+        this.id = id;
+        Age cloneAge = new Age();
+        cloneAge.setDay(age.getDay());
+        cloneAge.setMonth(age.getMonth());
+        cloneAge.setYear(age.getYear());
+        this.age = cloneAge;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Age getAge() {
```

Comment:
getAge should return instance of the age. It’s okay that age is mutable. Currently getAge always returns a new instance and that’s inconvenient in the code and inefficient.